### DISCUSSION TITLE
Where can we see when assignments are due?

### DISCUSSION THREAD

**Discussion by taylor-obrien** [earthlab-education/Earth-Analytics-2023-01-Intro#316] [! GitHub help] [answered]
I am wondering if I am missing somewhere in the repository/wiki that gives us information on when assignments are due. Also, is there anywhere in the repository that we can see all assignments that will be due for the semester? Not familiar with github, so not sure if I'm missing something!

**Comment by ellenalamont17** [comment, marked as answer]
This does not have the due dates listed, but on the wiki page, you can find a breakdown of the topics and assignments.

https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/wiki/Course-topics

I think there is generally some flexibility in the due dates. @nquarder mentioned in our discussion section today that reasonable progress should be made on assignments (discussion submitted and homework largely underway) by Tuesday after being assigned and that Friday is when assignments are expected to be completed. I had inquired about when should be have reviews completed, to which I was told he would check, but somewhere between 4-5 days after submission sounded realistic. Hope that helps a little.

**Reply by eculler** [reply]
That's right @ellenalamont17 - I will be downloading your work on Friday or Saturday, at which point I will email if I don't have your assignment. We try to be pretty flexible about the due dates in this program, as well as about resubmitting assignment, since things come up frequently when folks have family and work obligations (and sometimes the tech just doesn't work as expected). On the flip side - if you ever find yourself more than a week behind, it's really important for us to be in touch about that. It's easy to get very behind (we'll try to reach out, but you can, too).

You should be able to see the official due date if you take a look at the [Milestones](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/milestones). Each assignment going forward should have a milestone attached to it. I endeavor to also put correct due dates in Canvas and on GitHub classroom, but those are more for my purposes of keeping track of who has completed what.
